partcad/src/partcad/assembly_config.py

```
@@ -21,3 +22,8 @@ def normalize(name, config):
             config = {"type": "alias", "source": config}
 
         return ShapeConfiguration.normalize(name, config)
+
+    @staticmethod
+    def get_manufacturing_data(assembly) -> AssemblyConfigManufacturing:
```

Ruff (0.8.2)

Missing type annotation for function argument `assembly`

(ANN001)